src/sentry/plugins/providers/integration_repository.py

```diff
@@ -19,6 +19,17 @@ class IntegrationRepositoryProvider(object):
     def __init__(self, id):
         self.id = id
 
+    def get_client(self, integration_id):
+        if integration_id is None:
+            raise ValueError(self.name + 'requires an integration id.')
+
+        try:
+            integration_model = Integration.objects.get(id=integration_id)
+        except Integration.DoesNotExistError as error:
+            self.raise_error(error)
```

Comment:
I need to be able to throw an error here. @MeredithAnya or @macqueen let me know what would be a better idea here.